base/linalg/blas.jl

```diff
@@ -740,21 +740,26 @@ end
 ### trmv, Triangular matrix-vector multiplication
 
 """
-    trmv(side, ul, tA, dA, alpha, A, b)
+    trmv(ul, tA, dA, A, b)
 
-Returns `alpha*A*b` or one of the other three variants determined by `side` (`A` on left or
-right) and `tA` (transpose `A`). Only the `ul` triangle of `A` is used. `dA` indicates if
-`A` is unit-triangular (the diagonal is assumed to be all ones).
+Returns `op(A)*b`, where `op` is determined by `tA`
+(`N`, for nothing, `T`, for transpose `A`, and `C` for conjugate
```

Comment:
nothing is still confusing here. I think in blas it's supposed to mean "no transpose" but in terms of `op(A)` it's `op = identity`